docs-source/content/userguide/managing-domains/domain-events.md

```diff
@@ -34,11 +34,16 @@ The operator generates these event types in a domain namespace, which indicate t
  *  `NamespaceWatchingStarted`: The operator has started watching for domains in a namespace.
  *  `NamespaceWatchingStopped`: The operator has stopped watching for domains in a namespace. Note that the creation of this event in a domain namespace is the operator's best effort only; the event will not be generated if the required Kubernetes privilege is removed when a namespace is no longer managed by the operator.
 
+The operator also generates these event types in the operator namespace, which indicate the following:
+
+*  `StartManagingNamespace`: The operator has started to manage domains in a namespace.
```

Comment:
started to manage -> started managing (making this parallel with previous descriptions, such as `NamespaceWatchingStarted`)